### DISCUSSION TITLE
xDrip+ follower loosing readings

### DISCUSSION THREAD

**Discussion by PetarSambolic** [NightscoutFoundation/xDrip#2721] [Q&A] [answered]
Hi,
I am a xDrip+ follower and recently switched to Xiaomi phone (MIUI 13.0.17, Android 12 SKQ1.210908.001).
What I have discovered is that I am loosing readings starting every day at 11:30. The readings are appearing again when I go to xDrip.
I assume there is something wrong with either the phone or xDrip settings but I can not figure out what.
Could somebidy help me please?
Thanks in advance.

**Comment by Navid200** [comment, marked as answer]
See if disabling Google play protect will make a difference.
Search for how to disable Google play protect.

**Reply by PetarSambolic** [reply]
Thanks for the help, it worked!
